projects/ui-components/lf-user-feedback/feedback-image-upload/feedback-image-upload.component.ts

```diff
@@ -66,7 +66,9 @@ export class FeedbackImageUploadComponent {
   }
 
   private async tryReadAndValidateImageAsync(image: File | undefined): Promise<boolean> {
+    this.showLoader = true;
     if (!image) {
```

Comment:
use try/finally pattern, and refactor tryReadAndValidateImageAsync into multiple functions